gradle/repositories.gradle.kts

```diff
@@ -14,6 +14,7 @@ pluginManagement {
           includeModule("me.champeau.gradle", "japicmp-gradle-plugin")
           includeModule("com.gradle.publish", "plugin-publish-plugin")
           includeModule("com.github.ben-manes", "gradle-versions-plugin")
+          includeModule("org.jetbrains.kotlin.plugin.serialization", "org.jetbrains.kotlin.plugin.serialization.gradle.plugin")
```

Comment:
Sorry I missed that one. Do we need this? The serialization plugin should be on maven central these days: https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/plugin/serialization/org.jetbrains.kotlin.plugin.serialization.gradle.plugin/

Reply:
I removed it again in https://github.com/apollographql/apollo-kotlin/pull/4531/files, see what breaks 👀

Reply:
Version 1.6.10 (which we use during idea sync) is not on Maven central, but now I'm no longer sure this is needed because I removed it on my machine and it still works 😅

Reply:
Maybe you have configuration cache enabled 😅 . Ok I'll add it again in #4531 with a comment to remove it at some point